sklearn/tests/test_multiclass.py

```diff
@@ -704,6 +706,19 @@ def test_ecoc_fit_predict():
     assert len(ecoc.estimators_) == n_classes * 2
 
 
+def test_ecoc_predict_proba():
+    ecoc = OutputCodeClassifier(LinearSVC(random_state=0), random_state=0)
+    ecoc.fit(iris.data, iris.target).predict(iris.data)
+    proba = ecoc.predict_proba(iris.data)
+    # Test that the scores sum to 1
+    assert_almost_equal(np.sum(proba, axis=1), np.ones(proba.shape[0]))
+
+    # Regression test for the new proba-based predict against the old one
```

Comment:
I don't think we need this. Basically, this non-regression test is only check a "general" case. If something goes wrong, it would be on the corner case.

At least we have now the case where `y_prob.argmax(axis=1) == y_pred` which is something that we intend to achieve in common test.

Reply:
Looking at `OneVsRestClassifier()`, it only tests the output shape (which is reasonable I guess) and the said `y_prob.argmax(axis=1) == y_pred` equality. The latter seems redundant since `predict()` basically does nothing but `argmax(axis=1)` now.

As an alternative, ensuring the shortest distance yields the highest probability for a few cases comes to mind:
 - code size much smaller than the number of classes;
 - code size much bigger than the number of classes;
 - all equal distances.

Should we include those (or, perhaps, other possible cases)?